torch/fx/experimental/symbolic_shapes.py

```diff
@@ -255,30 +262,31 @@ def _nyi():
     'lt': lambda a, b: sympy.Lt(a, b),
     'le': lambda a, b: sympy.Le(a, b),
     'ge': lambda a, b: sympy.Ge(a, b),
+    'floor': lambda a: sympy.floor(a),
     'sym_float': lambda a: a,  # TODO: why can't I wrap with sympy.Float?
```

Comment:
so I guess you answered this TODO question, which is you can't wrap it because it expects Python literals

Reply:
yep, comment updated.